migrator/migrations/m_181_to_m_182_remove_default_scope_manager_role/migration_impl.go

```diff
@@ -0,0 +1,205 @@
+package m181tom182
```

Comment:
Probably needs rebasing like David mentioned in the other PR.

Reply:
indeed.